demo/index.html

```diff
@@ -18,64 +18,53 @@
 
   <title>iron-a11y-announcer demo</title>
 
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-snippet.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-pages-shared-styles.js"></script>
-  <script type="module" src="../../paper-button/paper-button.js"></script>
-  <script type="module" src="./x-announces.js"></script>
-  <script type="module" src="../iron-a11y-announcer.js"></script>
-  <!-- FIXME(polymer-modulizer):
-        These imperative modules that innerHTML your HTML are
-        a hacky way to be sure that any mixins in included style
-        modules are ready before any elements that reference them are
-        instantiated, otherwise the CSS @apply mixin polyfill won't be
-        able to expand the underlying CSS custom properties.
-        See: https://github.com/Polymer/polymer-modulizer/issues/154
-        -->
-    <script type="module">
-const $_documentContainer = document.createElement('template');
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
 
-$_documentContainer.innerHTML = `<custom-style>
-    <style is="custom-style" include="demo-pages-shared-styles">
-      input {
-        margin-right: 20px;
-      }
-    </style>
-  </custom-style>`;
+  <script type="module">
+    import '@polymer/iron-demo-helpers/demo-snippet.js';
+    import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+    import '@polymer/paper-button/paper-button.js';
+    import './x-announces.js';
+
+    import {html} from '@polymer/polymer/lib/utils/html-tag.js';
 
-document.body.appendChild($_documentContainer.content);
-</script>
+    const template = html`
+    <custom-style>
+      <style is="custom-style" include="demo-pages-shared-styles">
+        input {
+          margin-right: 20px;
+        }
+      </style>
+    </custom-style>
+    `;
+    document.body.appendChild(template.content);
+  </script>
 </head>
 <body>
-  <script type="module">
-const $_documentContainer = document.createElement('template');
-
-$_documentContainer.innerHTML = `<div class="vertical-section-container centered">
+  <div class="vertical-section-container centered">
     <p><b>Note</b>: in order to hear the announcements, be sure to turn on your favorite screen reader!</p>
 
     <h4>Announcer used directly on the main document</h4>
     <demo-snippet>
       <template>
         <input id="input" value="Pizza is delicious">
-        <paper-button raised="" onclick="_announce()">Announce</paper-button>
+        <paper-button raised="" onclick="window._announce()">Announce</paper-button>
```

Comment:
can you please check if this works without `window.`? If it does not, then can you put this in the README?

Reply:
Done! And added a known issues section